src/Compilers/CSharp/Portable/Syntax/RecordDeclarationSyntax.cs

```diff
@@ -32,15 +33,19 @@ public partial class SyntaxFactory
         }
 
         public static RecordDeclarationSyntax RecordDeclaration(SyntaxList<AttributeListSyntax> attributeLists, SyntaxTokenList modifiers, SyntaxToken keyword, SyntaxToken identifier,
-            TypeParameterListSyntax typeParameterList, ParameterListSyntax parameterList, BaseListSyntax baseList, SyntaxList<TypeParameterConstraintClauseSyntax> constraintClauses, SyntaxList<MemberDeclarationSyntax> members)
+            TypeParameterListSyntax? typeParameterList, ParameterListSyntax? parameterList, BaseListSyntax? baseList, SyntaxList<TypeParameterConstraintClauseSyntax> constraintClauses, SyntaxList<MemberDeclarationSyntax> members)
         {
```

Comment:
this was a bridge method for back compat.  But actually trying to use it produces terrible results.  For example, if you specify any members, you get:  `record C public string ToString();` (without braces).

Reply:
Consider adding a test to SyntaxFactoryTests.cs.